Validation/RecoTrack/python/GenParticleSelectionsForEfficiency_cff.py

```diff
@@ -10,6 +10,12 @@
     tipGP = cms.double(60),
     statusGP = cms.int32(1)
 )
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+phase1Pixel.toModify(GenParticleSelectionForEfficiency,minRapidityGP = -3.0, maxRapidityGP = 3.0)
+run3_common.toModify(GenParticleSelectionForEfficiency,minRapidityGP = -3.0, maxRapidityGP = 3.0)
+phase2_tracker.toModify(GenParticleSelectionForEfficiency,minRapidityGP = -4.5, maxRapidityGP = 4.5)
```

Comment:
It is probably good to update this file, even if `MultiTrackValidatorGenPs` has been largely unsupported (at least) for the past ~4.5 years (it might still work though).